ocp3/ocp3/GameManager.swift

```diff
@@ -94,8 +94,122 @@ class GameManager {
         team2 = makeTeam()
         team2?.teamInfo()
     }
+    ///////////////////////////////////Fonctions pour gerer les combats /////////////////////////////////////////
+    func attack(charAttacker: Character, charTarget: Character) {
```

Comment:
Maybe we can change the name `charAttacker` to `attacker` and `charTarget` to `target`? What do you think?

Reply:
Yes it' shorter.
But I have to add a label in func damage to avoid : target.damage(attacker: attacker)
Now it's : target.damage(from: attacker)

Reply:
Nice ! it's a really good idea to add a parameter name 👍